code/modules/surgery/organs/augments_arms.dm

```diff
@@ -349,36 +349,36 @@
 	slot = "l_arm_device"
 
 // lets make IPCs even *more* vulnerable to EMPs!
-/obj/item/organ/internal/cyberimp/arm/power_cord_apc
+/obj/item/organ/internal/cyberimp/arm/power_cord/apc
 	name = "APC-compatible power adapter implant"
 	desc = "An implant commonly installed inside IPCs in order to allow them to easily collect energy from their environment"
 	origin_tech = "materials=3;biotech=2;powerstorage=3"
-	contents = newlist(/obj/item/apc_powercord)
+	contents = newlist(/obj/item/powercord/apc)
 	requires_robotic_bodypart = TRUE
 
-/obj/item/organ/internal/cyberimp/arm/power_cord_apc/emp_act(severity)
+/obj/item/organ/internal/cyberimp/arm/power_cord/apc/emp_act(severity)
 	// To allow repair via nanopaste/screwdriver
 	// also so IPCs don't also catch on fire and fall even more apart upon EMP
 	if(emp_proof)
 		return
 	damage = 1
 	crit_fail = TRUE
 
-/obj/item/organ/internal/cyberimp/arm/power_cord_apc/surgeryize()
+/obj/item/organ/internal/cyberimp/arm/power_cord/apc/surgeryize()
 	if(crit_fail && owner)
 		to_chat(owner, "<span class='notice'>Your [src] feels functional again.</span>")
 	crit_fail = FALSE
 
 
-/obj/item/apc_powercord
```

Comment:
You made these two `/obj/item/powercord/apc` and `/obj/item/powercord/cell` subtypes of a ghost blank `/obj/item/powercord` type while leaving duplicated code on both, which isn't great

You should extract the common functionality between the two subtypes into this abstract parent type, and only leave the distinct differences on the subtypes - if you need some pointers on what this means and how to do it, feel free to ask on discord